docs/source/json-api/production-setup.rst

```diff
@@ -4,18 +4,10 @@
 Production Setup
 ################
 
-The vast majority of the prior documentation focuses on ease of testing and running
-the service in a dev environment. From a production perspective given the wide
-variety of use-cases there is far less of an established framework for deploying
-the *HTTP JSON API* server. In this document we would try to list some recommendations for
-production deployments.
+The vast majority of prior documentation focused on ease of testing and on setting up the service to run in a dev environment. From a production perspective, given the wide variety of use-cases, there is far less of an established framework for the deployment of an *HTTP JSON API* server. In this document we will make some recommendations for production deployments.
 
 The *HTTP JSON API* server is a JVM application that by default uses an in-memory backend.
-This in-memory backend setup is inefficient for larger datasets as for every query it
-ends up fetching the entire active contract set for the templates referenced in that query.
-For this reason for production setups at a minimum we recommend to use a database
-as a query store, this will allow for more efficient caching of the data to improve
-query performance. Details for enabling a query store are highlighted below.
+This in-memory backend setup is inefficient for larger datasets as every query ends up fetching the entire active contract set for all the templates the query references. For this reason, for production setups, we recommend, at a mimumum, that one use a database as a query store. This will allow for a more efficient caching of data and will improve query performance. Details for enabling a query store are highlighted below.
```

Comment:
```suggestion
This in-memory backend setup is inefficient for larger datasets, as every query fetches the entire active contract set for all the templates the query references. For production setups, therefore, we recommend that one use a database as a query store. This allows more efficient caching of data and improves query performance. Details for enabling a query store are provided below.
```
The "at a minimum" phrasing implies that there is more that could be done - is there, and should we describe it here? If not I recommend dropping that phrase.

Reply:
I'm ok with dropping at a minimum (if we don't plan to list what we really "at a maximum" would recommend. But I'd update the first sentence to: "This in-memory backend setup is inefficient for larger datasets, as every query fetches the entire active contract set for all of the templates the query references. "

Reply:
Sounds good.